tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -152,8 +156,8 @@ Delete VCH moved from its RP
     ${rand}=  Generate Random String  15
     ${dummyvm}=  Set Variable  anothervm-${rand}
     Set Suite Variable  ${tempvm}  ${dummyvm}
-    Log To Console  Create VM ${dummyvm} in %{TEST_RESOURCE}/%{VCH-NAME} net %{PUBLIC_NETWORK}
-    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.create -pool="%{TEST_RESOURCE}/%{VCH-NAME}" -net=%{PUBLIC_NETWORK} -on=false ${dummyvm}
+    Log To Console  Create VM ${dummyvm} in %{TEST_RESOURCE}/Resources/%{VCH-NAME} net %{PUBLIC_NETWORK}
```

Comment:
should we be using the `${pool}` variable here instead of the explicit path? That said, the `Resources` pool exists on ESXs as well, so another possibility is setting `TEST_RESOURCE` so that consistent path construction can be used irrespective of ESX or VC.

Reply:
Yes, I will add `${pool}` var but `TEST_RESOURCE` is a global var, so if I change / update it , won't it impact existing tests which are passing? @mhagen-vmware ?

Reply:
for now, let's just do the pool variable and please create a follow up issue to define the TEST_RESOURCE properly throughout all the code.

Reply:
Cool, I added https://github.com/vmware/vic/issues/7434 to take of it later.